core/templates/dev/head/galleries/Gallery.js

```diff
@@ -275,6 +284,7 @@ oppia.controller('SearchBar', [
       summary: ''
     }
   };
+  $scope.translationData = {}
```

Comment:
Actually -- I think I might have another idea. I'll make a comment in the new pull request.